packages/toast/src/Toast.ts

```diff
@@ -116,28 +116,44 @@ export class Toast extends FocusVisiblePolyfillMixin(SpectrumElement) {
         return this._variant;
     }
 
+    @property({ type: String })
+    public set label(label: string) {
+        this._label = label;
+        this.setAttribute('label', label);
+        this.requestUpdate('label', label);
+    }
+
+    public get label(): string {
+        return this._label;
+    }
```

Comment:
Can we not use a simple decorator here:

```
@property()
label = '';
```
And then avoid all the rest of the manual administration?

As far as naming, this isn't so much the `label` of the toast as much as the label of the icon. Should we have a more specific attribute/property name that represents that? Which ever way we go, it'll be good to have a JSDoc description as to how the value is leveraged within the element.